### PR TITLE
fail link to faq, updated to about

### PR DESCRIPTION
**Pull request by bucharitesh**
**Description of PR**

FAQ link changed to about

**Checklist**

- [ yes ] Compiles and passes lint tests
- [ yes ] Properly formatted
- [ yes ] Tested on desktop
- [ yes ] Tested on phone

**Screenshots**

Add relevant screenshots here

### CODE REVIEW

File: src/components/navbar.js [hunk 1]
```diff
@@ -100,7 +100,7 @@ function Navbar({pages, darkMode, setDarkMode}) {
             </span>
             <span>
               <Link to="/faq">
-                <Icon.HelpCircle {...activeNavIcon('/faq')} />
+                <Icon.HelpCircle {...activeNavIcon('/about')} />
```
Comment (bucharitesh) [inline src/components/navbar.js:103]: FAQ link broken changed to about page

**Comment by shuklaayush** [comment]
Do you mind replacing all the other links to /faq as well?

**Review by bucharitesh — COMMENTED** [review]
> Do you mind replacing all the other links to /faq as well?

the faq page isin't there, theres about page..... so i changed the one not working

**Comment by bee-rickey** [comment]
Hi have you had a look at the guidelines for contributing? 
https://github.com/covid19india/covid19india-react/blob/master/CONTRIBUTING.md
Please do not raise a PR before you raise an issue and the issue gets assigned. Thank you!

**Pull request merged**